test/param/loop/paramForLoopBoundSmall.good

```diff
@@ -0,0 +1 @@
+paramForLoopBoundSmall.chpl:1: error: Range in param for loop is too small for this count
```

Comment:
Wordsmithing, how about: `Count operator value exceeds range size`?  Or maybe even better `Count operator value '-5' exceeds range size of '3'`?

Reply:
Sir, is there a way we can use variables in `compilerError` function?

Reply:
> Sir, is there a way we can use variables in `compilerError` function?

Sir got the answer. I will update it now.